docs/13-walkthroughs/5-big-bang.md

```diff
@@ -0,0 +1,182 @@
+# Using Big Bang with Zarf
+
+## Introduction
+
+This walkthrough describes how to use Big Bang with Zarf for Air Gap deployments through the use of the Big Bang Zarf extension. If you are not familiar with Big Bang you can learn more about it here: https://p1.dso.mil/products/big-bang, but in short it is a DevSecOps platform that contains many useful tools for building, managing, and running software projects while adhering to the [United States Department of Defense DevSecOps Reference Design](https://public.cyber.mil/devsecops/).
+
+Zarf integrates with Big Bang through the use of an extension that simplifies the selection of Big Bang packages and the pulling of the required artifacts to deploy those packages in an Air Gap.
+
+### Known Issues
+
+The current version of this extension requires [some patches](../../examples/big-bang/config/patch-images.yaml) to the Big Bang values to function correctly, and is not fully integrated into the `zarf package remove` lifecycle.  We will be looking to remove these limitations in a future release.
+
+## System Requirements
+
+Big Bang requires a reasonably powerful `amd64` system that scales up with the number of components deployed.  We recommend at least `32 GB` of RAM and a high-speed internet connection to complete this walkthrough.
+
+To learn more about Big Bang's requirements in general, see their documentation: https://docs-bigbang.dso.mil/1.52.0/docs/prerequisites/minimum-hardware-requirements/
```

Comment:
The docs will re-direct to the latest tag using `latest`, so a potentially better URL here is <https://docs-bigbang.dso.mil/latest/docs/prerequisites/minimum-hardware-requirements/>